Rename/rokubi

jungerj opened the pull request:
# Renaming the Rokubi SDK
This adapts the changes from [this PR in the rokubimini_ethercat_sdk (RSL version)](https://bitbucket.org/leggedrobotics/rokubimini_ethercat_sdk/pull-requests/37/rename-rsl)

This shall not be merged before the aforementioned PR has been merged.

## Dependencies
To try this out: use the *rename/rsl* branch of the rokubimini_ethercat_sdk.

When we merge this PR the Rokubi changes will however be on master.

jungerj committed:
rename rokubi* ⟶ rokubi_rsl*

both in dependencies and head includes

jungerj committed:
explicit rokubi dependency not required anymore

zoenglinghou commented:
Maybe rename it in the README as well? Also which branch of Rokubimini are we using?

jungerj committed:
correct branches of dependencies

1) the new anydrive_sdk is already on master
2) the rokubis will also be on master before this PR is merged

jungerj commented:
Renamed dependencies in README to reflect the state we will be in once the rokubi PR has been merged.
